addon/models/property-group.js

```diff
@@ -25,6 +28,12 @@ export default class PropertyGroupModel {
       formGraph
     );
     this.rdflibHelp = store.any(uri, FORM('help'), undefined, formGraph);
+
+    let rdfLevel = store.any(uri, EXT('level'), undefined, formGraph);
```

Comment:
I've used an `ext` prefix for now since I wasn't sure which predicate uris to use. Suggestions?

Should we solve this in another way?

Reply:
Maybe we should pass the data through `form:options` instead?

Reply:
I went with the form:options version.